sdk/bootstrap/build.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# exit immediately on failure
+set -e
```

Comment:
I'd add this but there an explicit check for `$GOPATH` which fails in the unset case:
```
if [ -z "$GOPATH" -o -z "$(which go)" ]; then
  echo "Missing GOPATH environment variable or 'go' executable. Please configure a Go build environment."
  exit 1
fi
```

Result with `set -eu` (no pretty error):
```
./build.sh: line 9: GOPATH: unbound variable
```